src/client/opamClient.ml

```diff
@@ -155,6 +155,9 @@ let with_switch_backup command f =
   let t = OpamState.load_state command OpamStateConfig.(!r.current_switch) in
   let file = OpamPath.Switch.backup t.root t.switch in
   OpamFilename.mkdir (OpamPath.Switch.backup_dir t.root t.switch);
+    OpamFile.Environment.write
+      (OpamPath.Switch.environment t.root t.switch)
+      (OpamState.compute_env_updates t);
```

Comment:
Why is this needed here ? It's not backup-related and the file should already be there, no ?